ddtrace/internal/wrapping/context.py

```diff
@@ -0,0 +1,453 @@
+import sys
```

Comment:
docstring with usage/rationale would be great.

Reply:
technical background on how it works could be nice too, especially for people who struggle to read the bytecode.